test-data/local/ECLIPSE/simple/SIMPLE.DATA

```diff
@@ -0,0 +1,59 @@
+RUNSPEC
+START
+1 JAN 2017 /
+
+DIMENS
+ 10 10 10 /
+OIL
+UNIFOUT
```

Comment:
No syntax erorr, but I would not separate the line with OIL and the line with WATER with UNIFOUT. Humans need to view line 7 and 9 together to figure out which phase configuration is active.

Reply:
Fixed